Description: Upstream changes introduced in version 0.02
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 classicmenu-indicator (0.02) natty; urgency=low
 .
   * cmindicator.py:
     - on_menuitem_activate: don't wait for the command to finish (LP: #792760)
 .
 The person named in the Author field signed this changelog entry.
Author: Florian Diesch <devel@florian-diesch.de>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/792760

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- classicmenu-indicator-0.02.orig/Makefile
+++ classicmenu-indicator-0.02/Makefile
@@ -1,6 +1,6 @@
 NAME=classicmenu-indicator
-VERSION=0.01
-DEBVERSION=0.01
+VERSION=0.02
+DEBVERSION=0.02
 PPA=diesch/testing
 
 .PHONY: clean deb sdist ppa deb
